WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/WatchDogView.java

```diff
@@ -216,4 +245,33 @@ public void setFocus() {
 		parent.setFocus();
 	}
 
+	private PieDataset createPerspectiveViewPieDataset() {
+		double divisor = perspectiveDebug + perspectiveJava + perspectiveOther;
+		final DefaultPieDataset result = new DefaultPieDataset();
```

Comment:
In every method that creates DataSet, resulting object (DefaultCategoryDataset, DefaultPieDataSet) was marked as final. Not really sure why this is the case, but I wanted to follow the same logic.